eventuate-tram-messaging-proxy-service/src/test/resources/application.properties

```diff
@@ -1 +1,28 @@
 eventuate.http.proxy.zookeeper.connection.string=${DOCKER_HOST_IP:localhost}:2181
+eventuatelocal.kafka.bootstrap.servers=${DOCKER_HOST_IP:localhost}:9092
+
+eventuate.subscription.message.1.channels=orders,order-history
+eventuate.subscription.message.1.url=http://${DOCKER_HOST_IP:localhost}:8081/messages/1
+
+eventuate.subscription.message.2.channels=customers,customer-history
+eventuate.subscription.message.2.url=http://${DOCKER_HOST_IP:localhost}:8081/messages/2
```

Comment:
`eventuate.subscription.message.2.url` should not include the `subscriberId`, i.e. '2'.
The proxy should automatically include that in the path.

Reply:
ok, I will fix